tests/unit/test-class-wc-payments-localization-service.php

```diff
@@ -172,6 +173,32 @@ public function test_transient_data_set() {
 		$this->assertTrue( is_array( get_transient( WC_Payments_Localization_Service::WCPAY_LOCALE_INFO_TRANSIENT ) ) );
 	}
 
+	public function test_currency_switches_for_a_country_is_not_effective_until_timestamp() {
+		add_filter( 'transient_' . WC_Payments_Localization_Service::WCPAY_CURRENCY_FORMAT_TRANSIENT, '__return_false' );
+		add_filter( 'transient_' . WC_Payments_Localization_Service::WCPAY_LOCALE_INFO_TRANSIENT, '__return_false' );
+		add_filter(
+			'wcpay_mock_time',
+			function() {
+				return strtotime( '2022-12-31 23:59:59' );
```

Comment:
This test is failing on GH but passes locally. Maybe having the mock time only one second away from the effective timestamp is causing issues with the tests running on GH? 

I'm going to change the timestamp to be further away from `2023-01-01 00:00:00` to see if that resolves the issue for the tests running on GH.

Reply:
Pushing back the mock timestamp by ten minutes and ten seconds didn't resolve it. I wonder if it is a timezone thing? I'll test it again by moving it back an entire day.